### DISCUSSION TITLE
wasm support

### DISCUSSION THREAD

**Discussion by paganaye** [irgaly/kottage#236] [Q&A]
There is no mention in the README.md, does this work with wasmJs?

**Comment by irgaly** [comment]
Not supported yet, because wasmJs is not available when I released Kottage.

Kotlin Wasm is Alpha now https://kotlinlang.org/docs/wasm-overview.html , so I'll try adding wasmJs() target. Please wait for a little while 😄 

At this time, Kottage wasmJs() on browser will use IndexedDB.

**Comment by irgaly** [comment]
We need to wait SQLDelight supporting wasmJs() target.

* cashapp/sqldelight/pull/4965
